File: loopsum/loopsum.jl
using Random
using Printf

try
    using BenchmarkTools
catch
    import Pkg
    Pkg.add("BenchmarkTools")
    Pkg.update("BenchmarkTools")
    using BenchmarkTools
end


function loopsum(n)
    x = 0.
    for i=1:n
        x += rand()
    end
    x
end

o = @benchmark loopsum(n) setup=(n=10^6)
print("Mean time: ", time(o) / 1e6, "ms\n")

File: loopsum/loopsum.py
import timeit

setup = """
import random

def loopsum(n):
    x = 0.
    for i in range(n):
        x += random.random()
    return x
"""

num_runs = 50
time = timeit.timeit("loopsum(10**6)", setup=setup, number=num_runs)
print("Mean time: {:,.3f} ms".format(time * 1e3))
